src/SignalR/common/Protocols.Json/src/Microsoft.AspNetCore.SignalR.Protocols.Json.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Implements the SignalR Hub Protocol using System.Text.Json.</Description>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <IsAspNetCoreApp>false</IsAspNetCoreApp>
```

Comment:
Is this temporary? Or is it really not part of the plans to JSON protocol support into the shared framework?

Reply:
Temporary

Reply:
👍 Ok, just wanted to make sure.

Reply:
So it isn't in the shared framework?

Reply:
I guess there is no reason not to add it now.

Reply:
Fine by me. Any reason we can't also remove the NewtonsoftJson version from the shared framework as a part of this PR?

Reply:
Because we haven't decided to do it yet.